### ISSUE TITLE
Bombs for the enemy

### ISSUE THREAD

**Issue by Necrofox91** [MegaMek/mekhq#924]
So this came up in Joker's stream, just watched it. And I searched for issues and I didn't find it.
Basically the idea is for enemy air assets to come preloaded with bombs. Maybe not always, make it scenario dependent... like not in a chase since they are responding with fast movers, but yes in base assaults and stand up fights. Plus enemy bombers are more likely to have bombs than the rest and interceptors just don't. A lot of thought will need to be given to the different combinations of enemy assets, the current mission and different chances and kinds of bombs, plus the era and availability and Clan vs IS.
So here it is. thoughts on this are welcome.

**Comment by NickAragua** [comment]
I assume you're talking about bot-controlled aircraft generated as part of AtB scenarios. This will be looked at as part of some on-going work, so look for more news on the front "as it happens".

**Comment by Necrofox91** [comment]
Yep exactly. Just saw it in the stream and I realized that it also was true for me so I did some searching and decided to report. It isn't so much a bug, more of an enhancement than anything else, but even if I'm piling up more work for you guys, when I report things properly and when I don't too...
I hope having things like this reported and tested and on a convenient list of things to work for instead of just having to figure it out all by yourself helps in some way.
Though with my reporting and commenting tendencies until now...
Well... Hope I help more than I hinder at least?
